src/mdlComponentHandler.js

```diff
@@ -86,6 +86,15 @@ var componentHandler = (function() {
    * the element to.
    */
   function upgradeElementInternal(element, jsClass) {
+    if (jsClass === undefined) {
+      for (var i = 0; i < registeredComponents_.length; i++) {
```

Comment:
Actually the JS will only be ran starting on IE 10 (which his why using `.classList` is OK) on IE9 MDL is CSS only.
For now I've been pushed to use `forEach` in most of my Commits ;)